packages/calcite-components/src/custom-theme/button.ts

```diff
@@ -0,0 +1,15 @@
+import { html } from "../../support/formatting";
+
+export const button = (args: { kind?: string; appearance?: string }): string =>
+  html`<calcite-button
+    ${Object.entries(args)
```

Comment:
I find this a bit tricky to read (at least in a single pass). Could you refactor this to improve readability? A function might be able to be shared between the attribute and content setting.

Reply:
Let me know if this update makes more sense